### PR TITLE
Create reverseproxy.md

### PR DESCRIPTION
**Pull request by patski123**

### CODE REVIEW

File: docs/self-hosted-apps/reverseproxy.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+**Project Page:** [containous/traefik](https://github.com/containous/traefik)
+
+**Docker Hub:** [traefik](https://hub.docker.com/_/traefik)
+
+### Reviews and Related Links
+   * 
+   *
+
+
+## Linuxserver.io's Nginx Let's Encrypt Reverse Proxy
```
Comment (RealOrangeOne) [inline docs/self-hosted-apps/reverseproxy.md:18]: This section should really be about Nginx itself, and then list the LSIO container underneath, along with why you might want to use it

File: docs/self-hosted-apps/reverseproxy.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Self Hosted Reverse Proxy
+
```
Comment (py-crash) [inline docs/self-hosted-apps/reverseproxy.md:2]: Can you add some explanation about what is a reverse proxy and why should you use one?

File: docs/self-hosted-apps/reverseproxy.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+**Website:** [https://traefik.io](https://traefik.io)
+
+**Project Page:** [containous/traefik](https://github.com/containous/traefik)
+
+**Docker Hub:** [traefik](https://hub.docker.com/_/traefik)
+
+### Reviews and Related Links
+   * 
+   *
```
Comment (py-crash) [inline docs/self-hosted-apps/reverseproxy.md:15]: Please add some links or mark the PR as a draft if it's not a finished article

File: docs/self-hosted-apps/reverseproxy.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+
+## Linuxserver.io's Nginx Let's Encrypt Reverse Proxy
+Organizr aims to be your one stop shop for your Servers Frontend
```
Comment (pagdot) [inline docs/self-hosted-apps/reverseproxy.md:19]: This line seems to be copy-pasted but hasn't got updated for nginx

**Comment by IronicBadger** [comment]
Closing due to lack of activity. Please note we have changed to `master` as the default branch for future PRs. This article will be very useful so please consider opening a new PR with the comments from this one incorporated.

Thanks!

**Pull request closed without merging**